### PR TITLE
Forms Automated Test - LPS-130510 | master

### PR DESCRIPTION
**Pull request by lima-rafa**

### CODE REVIEW

**Comment by brittneyq** [comment]
Hi @lima-rafa, it looks like you pushed two test cases, could you change it so only the test associated with the ticket is committed?

**Comment by brittneyq** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase [hunk 1]
```diff
@@ -5627,13 +5627,45 @@ definition {
...
+		FormsAdminNavigator.gotoBack();
+
+		FormsAdminNavigator.gotoEditFormByName(formName = "Form 1");
+
+		AssertTextEquals(
+			locator1 = "Form#NAME_FIELD",
+			value1 = "Form 1");
+		AssertTextEquals(
+			locator1 = "Form#DESCRIPTION_FIELD",
+			value1 = "Element Set Description");
```
Comment (brunofariasdeo) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase:5662]: We also have a macro that asserts the form description, which is `Form.viewFormDescription`.
In this case, you would use something similar to `Form.viewFormDescription(formDescription = "Form Description");`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase [hunk 2]
```diff
@@ -5627,13 +5627,45 @@ definition {
...
+		AssertElementPresent(
+			fieldLabel = "Text",
+			locator1 = "FormViewBuilder#FIELD_LABEL");
```
Comment (brunofariasdeo) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase:5665]: There is a macro which you can use in order to check if the field label is already rendered, and it is the `FormFields.viewTextLabel`. 
You can use it by passing the field label as a parameter, something like `FormFields.viewTextLabel(fieldLabel = "Text")`.

**Review by brunofariasdeo — CHANGES_REQUESTED** [review]

**Review by brunofariasdeo — APPROVED** [review]

**Comment by brunofariasdeo** [comment]
Very good, great job, @lima-rafa :)

**Comment by brunofariasdeo** [comment]
ci:test:sf

**Comment by brunofariasdeo** [comment]
ci:test:relevant

**Comment by brunofariasdeo** [comment]
ci:forward

**Comment by brunofariasdeo** [comment]
ci:test:sf

**Comment by brunofariasdeo** [comment]
Hi @lima-rafa ,
There was an sf issue that I didn't quite understood what was going on, but followed some steps and then it worked out haha.
I've resent the PR [here](https://github.com/liferay-forms/liferay-portal/pull/383).
Thanks!

**Pull request closed without merging**